.github/workflows/tests_gpu_python.yml

```
@@ -174,76 +168,53 @@ jobs:
       - name: Install Stable PennyLane
         if: inputs.pennylane-version == 'stable'
         run: |
-          cd main
           python -m pip uninstall -y pennylane && python -m pip install -U pennylane
 
       - name: Build and install package
         env:
           CUQUANTUM_SDK: $(python -c "import site; print( f'{site.getsitepackages()[0]}/cuquantum')")
         run: |
-          cd main
           rm -rf build
           PL_BACKEND=lightning_qubit python scripts/configure_pyproject_toml.py || true
-          PL_BACKEND=lightning_qubit SKIP_COMPILATION=True python -m pip install . -vv
+          SKIP_COMPILATION=True python -m pip install . -vv
```

I know this seems redundant, but could we keep the `PL_BACKEND` as before? This helps the dependency readability of the action. 
```suggestion
          PL_BACKEND=lightning_qubit SKIP_COMPILATION=True python -m pip install . -vv
```

---

Really redundant especially because we set the backend in the line above but I can add it.